cluster/grpc_rpc_server.go

```diff
@@ -52,8 +52,9 @@ func NewGRPCServer(config *config.Config, server *Server, metricsReporters []met
 
 // Init inits grpc rpc server
 func (gs *GRPCServer) Init() error {
+	address := gs.config.GetString("pitaya.cluster.rpc.server.grpc.address")
```

Comment:
won't address always be 0.0.0.0?

Reply:
No, in the tests when starting the I pass the `-address` flag with `localhost` value.

Reply:
oh ,oks